test/crossrunner/run.py

```diff
@@ -243,7 +243,11 @@ def ensure_socket_open(sv, port, test):
                 # kill them off; if we didn't kill them off, something else
                 # happened (crashed?)
                 if test.server.stop_signal != 0:
-                    if sv.killed or sv.returncode > 0:
+                    # for bash scripts, 128+N is the exit code for signal N, since we are sending
+                    # DEFAULT_SIGNAL=1, 128+1 is the expected err code
+                    # http://www.gnu.org/software/bash/manual/html_node/Exit-Status.html
+                    allowed_return_code = set([-1, 0, 128+1])
```

Comment:
Not sure why its says "./test/crossrunner/run.py:249:58: E226 missing whitespace around arithmetic operator" but it does